backend/backend/settings.py

```diff
@@ -130,4 +130,8 @@
 
 CORS_ORIGIN_WHITELIST = (
     'http://localhost:3000',
-)
\ No newline at end of file
+)
+
+REST_FRAMEWORK = {
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination'
+}
```

Comment:
Don't think you need this, this is to set up the LimitOffsetPagination as the pagination class for all your viewsets. Don't think it hurts to leave it, but just so you know that you don't need it to have the pagination in the ToDoViewset

Reply:
Good catch